molecule/command/init/scenario.py

```diff
@@ -137,7 +142,7 @@ def _default_scenario_exists(ctx, param, value):  # pragma: no cover
 @click.option(
     '--role-name',
     '-r',
-    required=True,
+    required=False,
     callback=_role_exists,
```

Comment:
@decentral1se I hope this solved the problem you remarked regarding calling `molecule init scenario`. Now you can freely call it with `molecule init scenario -s foo` from inside the role directory.

Reply:
YES. This is great.